src/command_line/main.cc

```diff
@@ -36,56 +37,72 @@ int main(int argc, char** argv) {
   using CostModelLD = cost_model::UnitCostModelLD<Label>;
   using LabelDictionary = label::LabelDictionary<Label>;
 
-  // Runtime measurement variables (rusage).
-  rusage before_rusage;
-  rusage after_rusage;
-  timeval runtime_utime;
-  timeval runtime_stime;
-  unsigned int runtime;
+  /////// Runtime measurement should be limited to tests, and use portable libraries if available
+  //// Runtime measurement variables (rusage).
+  //rusage before_rusage;
+  //rusage after_rusage;
+  //timeval runtime_utime;
+  //timeval runtime_stime;
+  //unsigned int runtime;
```

Comment:
Let's delete obsolete code here and later in the file. With @thuetter we agreed not to do the runtime and memory measurements in this library.